subsys/sal/sid_pal/src/sid_mfg_storage.c

```diff
@@ -60,48 +60,60 @@ struct sid_pal_mfg_store_value_to_address_offset {
 static void ntoh_buff(uint8_t *buffer, size_t buff_len);
 static uint32_t default_app_value_to_offset(int value);
 static off_t checked_addr_return(off_t offset, uintptr_t start_address, uintptr_t end_address);
-static off_t value_to_offset(sid_pal_mfg_store_value_t value, uintptr_t start_address, uintptr_t end_address);
+static off_t value_to_offset(sid_pal_mfg_store_value_t value, uintptr_t start_address,
+			     uintptr_t end_address);
 
 // *INDENT-OFF*
```

Comment:
This comment is only relevant for uncrustify, it should be removed or replaced with clang-format comment to disable formatting

Reply:
Done